firefox-ios/Client/Frontend/Browser/TabTrayButtonExtensions.swift

```diff
@@ -32,11 +32,6 @@ class PrivateModeButton: ToggleButton, PrivateModeUI {
         tintColor = isPrivate ? theme.colors.ecosia.backgroundPrimary : theme.colors.ecosia.textPrimary
         imageView?.tintColor = tintColor
 
-        // Ecosia: Modify background layer
-        backgroundLayer.backgroundColor = isPrivate
-            ? theme.colors.ecosia.privateButtonBackground.cgColor
-            : UIColor.clear.cgColor
-
```

Comment:
Leaving it as Firefox intended provided a better result (see [this comment](https://ecosia.atlassian.net/browse/MOB-3396?focusedCommentId=104279))